CMakeLists.txt

```diff
@@ -106,8 +106,8 @@ set_target_properties(libhiredis PROPERTIES
 # Define redis++ as an external project
 ExternalProject_Add(redis++
     GIT_REPOSITORY https://github.com/sewenew/redis-plus-plus.git
-    GIT_TAG 1.3.10
-    GIT_SHALLOW 1
+    GIT_TAG e30d4c4c557568b01d5adce9df2714de8d3921c2
+    GIT_SHALLOW 0
```

Comment:
Not wrong, just curious: is there a reason we are grabbing this specific commit over release `1.13.4`? Scrolling through the additional commits from that point I see they updated some of their CMake build, but I didn't see anything that looked too problematic?

Reply:
The CMake build is problematic. CMake now errors out unless `CMAKE_POLICY_MINIMUM_VERSION` is set to 3.5 or higher. There is no way to circumenvent it in our build pipeline, therefore I grabbed the firtst commit where it was fixed for `redis++`.